x/tokenomics/keeper/keeper_settle_pending_claims_test.go

```diff
@@ -302,8 +317,9 @@ func (s *TestSuite) TestSettlePendingClaims_ClaimExpired_ProofRequiredAndNotProv
 	expectedClaimExpiredEvent := expectedClaimExpiredEvents[0]
 	require.Equal(t, tokenomicstypes.ClaimExpirationReason_PROOF_MISSING, expectedClaimExpiredEvent.GetExpirationReason())
 	require.Equal(t, s.numRelays, expectedClaimExpiredEvent.GetNumRelays())
-	// TODO(@red-0ne, #781): Ensure other claim expiration event fields are validated once added
-	// TODO_FOLLOWUP: Add NumEstimatedComputeUnits and ClaimedAmountUpokt
+	require.Equal(t, s.numClaimedComputeUnits, expectedClaimExpiredEvent.GetNumClaimedComputeUnits())
+	require.Equal(t, s.numEstimatedComputeUnits, expectedClaimExpiredEvent.GetNumClaimedComputeUnits())
```

Comment:
```suggestion
	require.Equal(t, s.numEstimatedComputeUnits, expectedClaimExpiredEvent.GetNumEstimatedComputeUnits())
```

Reply:
I'm assuming the above was copypasta, do we also want an assertion that `s.numEstimatedComputeUnits == s.numClaimedComputeUnits`?